TracksDemo/TracksDemo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -33,8 +33,8 @@
         "repositoryURL": "https://github.com/getsentry/sentry-cocoa",
         "state": {
           "branch": null,
-          "revision": "dc00a3673e3d313e41326bd53802f757defd0eee",
-          "version": "7.28.0"
+          "revision": "2479b6f7ff69b66bcdea82184d097667e63828ed",
+          "version": "8.5.0"
```

Comment:
@salimbraksa I noticed you reverted this change in d23173c215b0727861800a5fbbbaa25ca50c14da, but your Xcode must have snuck it in again. 

FWIW, I think it's appropriate to keep it, even though it looks out of place in this PR. It should have been part of https://github.com/Automattic/Automattic-Tracks-iOS/pull/245, which updated Sentry in the main `Package.resolved` file.

https://github.com/Automattic/Automattic-Tracks-iOS/blob/5e42ccaee51d6cd7c9e0f64f2a939a6ee32ba44b/Package.resolved#L28

This file is irrelevant for consumers of the library, and only used by Xcode to manage the versions in the demo app workspace.